src/components/Referrals/referralsHelper.js

```diff
@@ -13,6 +14,12 @@ import { getRootUrl } from "lib/url";
 
 export const REFERRAL_CODE_REGEX = /^\w+$/; // only number, string and underscore is allowed
 export const REGEX_VERIFY_BYTES32 = /^0x[0-9a-f]{64}$/;
+const MAX_DISCOUNT_SHARE = 10000; // 100%
```

Comment:
i think it's okay to use BASIS_POINTS_DIVISOR? discount/rebates are measured in basis points

Reply:
Refactored!